.travis.yml

```diff
@@ -18,7 +18,7 @@ stages:
   - name: release
     if: (branch = master AND type = push) OR (tag IS present)
   - name: upload-launcher
-    if: (branch = master AND type = push) OR tag IS present
```

Comment:
Good catch. I this change was supposed to be applied to the update-docker-images stage in order to support snapshot images.

Reply:
Yep, np!

Reply:
Ah, my bad, I opened https://github.com/almond-sh/almond/pull/343 to re-instate it.